server/src/main/java/io/crate/metadata/Functions.java

```diff
@@ -118,6 +118,9 @@ public Iterable<Signature> signatures() {
     }
 
     public void setUDFs(Map<FunctionName, List<FunctionProvider>> functions) {
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug("UDFs are loaded");
+        }
```

Comment:
```suggestion
```

The message is missleading as this is also called to update the registered udfs and I don't get why we'd need this logged